### PR TITLE
Issue 73: considerations on sharing a repo

### PR DESCRIPTION
**Pull request by jenmalloy**

### CODE REVIEW

File: topics/considerations-sharing-repo.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[[ccg-considerations-sharing-repo]]
+= Considerations for sharing a repository with engineering
+
+If you are considering sharing a repository with engineering, that is, for the software code and documentation to reside in the same upstream repository, here are a few things you will want to keep in mind.
+
+== Space considerations
```
Comment (ncbaratta) [inline topics/considerations-sharing-repo.adoc]:
```suggestion
.Space considerations
```

File: topics/considerations-sharing-repo.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+The size of the combined repository will be large, so if limited space is an issue, keep this factor in mind.
+
+== Rebase often
```
Comment (ncbaratta) [inline topics/considerations-sharing-repo.adoc]:
```suggestion
.Rebase often
```

File: topics/considerations-sharing-repo.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+When working in the same repository as engineering, rebasing frequently is often a necessity because engineering is making changes too. Before making any changes to the documentation you’ll want to rebase.
+
+== Upstream shared repository branches and downstream docs repository branches
```
Comment (ncbaratta) [inline topics/considerations-sharing-repo.adoc]:
```suggestion
.Upstream shared repository branches and downstream docs repository branches
```

File: topics/considerations-sharing-repo.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+Although you’ll be sharing the upstream repository with engineering, you’ll also need a downstream docs repository. Since more than likely the upstream repository branching structure will not match exactly the downstream docs repository branching, you will need to sync up the content. The easiest way to sync up content is to use the git export script.  
+
+Also, figuring out a branching strategy ahead of time will save plenty of headaches in the long run.
+
+== Use the git export script
```
Comment (ncbaratta) [inline topics/considerations-sharing-repo.adoc]:
```suggestion
.Use the git export script 
```

File: topics/considerations-sharing-repo.adoc [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+== Upstream shared repository branches and downstream docs repository branches
+Although you’ll be sharing the upstream repository with engineering, you’ll also need a downstream docs repository. Since more than likely the upstream repository branching structure will not match exactly the downstream docs repository branching, you will need to sync up the content. The easiest way to sync up content is to use the git export script.  
+
+Also, figuring out a branching strategy ahead of time will save plenty of headaches in the long run.
+
+== Use the git export script 
+Another tip is to use the git export script to sync up content:
+https://github.com/redhat-documentation/tools/tree/master/git-export 
+
+== Use attributes
```
Comment (ncbaratta) [inline topics/considerations-sharing-repo.adoc]:
```suggestion
.Use attributes
```

File: topics/considerations-sharing-repo.adoc [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+.Use the git export script 
+Another tip is to use the git export script to sync up content:
+https://github.com/redhat-documentation/tools/tree/master/git-export 
+
+.Use attributes
+In general, using attributes will facilitate ease of use between upstream docs and downstream docs. One of the most useful is for the product name itself: {ProductName}. Other useful attributes include those for any version numbers used in the docs or any other product names.  For more general information about how to use attributes, see https://redhat-documentation.github.io/community-collaboration-guide/#ccg-asciidoc-attributes.
+
+In particular, using a variable for the upstream directory in the downstream docs repo is helpful. For example, define the following variable in the downstream docs repository: `:UpstreamDir: upstream/documentation`
+Then, in the downstream master.adoc files, the include looks like this: `include::{UpstreamDir}/assemblies/assembly-installing.adoc[leveloffset=+1]`
```
Comment (pmkovar) [inline topics/considerations-sharing-repo.adoc]: `:UpstreamDir: upstream/documentation` and `include::{UpstreamDir}/assemblies/assembly-installing.adoc[leveloffset=+1]` should be separate code blocks. 

Separate paragraphs with a blank line.

File: topics/considerations-sharing-repo.adoc [hunk 7]
```diff
@@ -0,0 +1,26 @@
+[[ccg-considerations-sharing-repo]]
```
[the remark this answers is not in the document]
Comment (ncbaratta) [reply topics/considerations-sharing-repo.adoc]: @pmkovar , we were talking about that and debating where to put it specifically. Do you have any ideas?

File: topics/considerations-sharing-repo.adoc [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+= Considerations for sharing a repository with engineering
+
+If you are considering sharing a repository with engineering, that is, for the software code and documentation to reside in the same upstream repository, here are a few things you will want to keep in mind.
+
+.Space considerations
```
[the remark this answers is not in the document]
Comment (ncbaratta) [reply topics/considerations-sharing-repo.adoc]: The disk space thing came from the internal tools team ... but in theory, I do agree with you.

File: topics/considerations-sharing-repo.adoc (hunk 7 above)
[the remark this answers is not in the document]
Comment (pmkovar) [reply topics/considerations-sharing-repo.adoc]: Could be a part of this section I think: https://redhat-documentation.github.io/community-collaboration-guide/#ccg-project-structure

**Pull request merged**